### ISSUE TITLE
Less accuracy on local machine compare to Colab

### ISSUE THREAD

**Issue by helper423** [mrdbourke/pytorch-deep-learning#841]
Hi, when I try run same training code on my local machine and on Colab, on Colab model accuracy higher than on local machine, on both installed pytorch 2 and cuda 1.2.1, anyone know how to solve this problem?

**Comment by Lemonade678** [comment]
Can you show me code that have a problem because how would i know if dont see the code snipplet itself
